content/app/development/ux/images/_index.en.md

```diff
@@ -66,24 +66,25 @@ The image can also have separate sources for different languages. The default so
 
 ## Hosting images from apps
 
-If the image should be loaded from the app, you need to set up static hosting of files in the application.
+If the image should be loaded from the app, you need to set up static hosting of files in the application (if you have an older app).
```

Comment:
How about rather referring to the specific app version where it no longer is required? I guess this would be the date the template was made available in Altinn Studio. Let me know the PR that fixed this, and I'll get you the correct date.

Reply:
Great suggestion. And 👍 for offering help to figure out the date. The link is in the PR description, and as it is not merged (yet) it is hard to know the date. We should update when a date/version is available.

Reply:
@ivarne 
Could you update this text to something like 
```
If the image should be loaded from the app, you need to set up static hosting of files in the application. This is automatically set up for applications created from template version 3.19.1 and newer. For older applications, you should follow the steps below.
```
After that, I think we can merge this.

Reply:
Great!

The version is wrong, because this should be tied to the version of altinn-studio that was used to initialize the app, not the version of app-frontend. I'm still in the dark when the next version of sudio will be deployed, but I think this language might work.

```suggestion
If the image should be loaded from the app, you need to set up static hosting of files in the application. This is automatically set up for applications created after december 2021. For older applications, you should follow the steps below.
```

Reply:
This looks good enough to me. WDYT @acn-sbuad ?